src/components/Nav/Nav.scss

```
@@ -0,0 +1,107 @@
+@import '../../styles/baseStyle/font.scss';
+@import '../../styles/baseStyle/size.scss';
+@import '../../styles/baseStyle/colors.scss';
+
+.navHead {
+  box-shadow: 0 3px 7px 0 rgb(0 0 0 / 10%);
+  .navUpper {
+    height: 40px;
+    background-color: #fafafa;
+    .navUpperFixedWidth {
+      position: relative;
+      width: $widthSize;
+      height: 100%;
+      margin: 0 auto;
+      nav {
+        display: flex;
+        justify-content: flex-end;
+        line-height: 40px;
+        a {
+          font-size: $smallFontSize12;
+          color: $colorBlack111;
```

```suggestion
          font-size: $fontSize12;
          color: $colorBlack111;
```
- smallFontSize -> 그냥 fontSize 해도 되지 않을까요?
- black은 당연히 color이므로 black 그냥 쓰시면 될거 같고 111의 네이밍 이유도 궁금해요 어떤 의도인가요?

---

안녕하세요, 종택님!
- `smallFontSize`는 폰트 사이즈가 여러 개라서 `title`, `medium`, `default`, `small`로 크게 나누었습니다.
그리고 그 안에 포함된 항목들이 사이즈만 다르고 특징이 없어서 `px` 사이즈를 변수 끝에 기재하였습니다.
아니면 이 또한 명확하지 않으므로 종택님 말씀대로 `fontSize64`, `fontSize14`로 수정하는 것도 생각해 보았는데 괜찮을까요?
- `color`같은 경우에는 닥터마틴 사이트의 경우 유사한 black이어도 hex가 다른 경우가 있어 뒤에 hex를 작성하였습니다.
`color`라는 이름을 제외하고 `$black111`로 하는 것도 네이밍에 문제가 없을지 여쭤보고 싶습니다.

---

@KH2Wone 
- 그런 목적이라면 차라리 font-small, font-sm, small, sm, font-title 등의 네이밍이 더 좋을 것 같습니다. 구체적인 픽셀이 들어간다면 네이밍을 해주는 의미가 많이 퇴색되는거 같아서요
- color 역시 구체적인 hex 값을 적어주는 것보다 light-gray, gray, deep-gray 등의 이름으로 구분해주거나 아니면 gray100, gray200, gray300 등 진하기를 숫자로 표현하는 방식이 나아보여요

---

희원님께서 변경하신 이력에 맞게 변수명 재정립 진행했습니다!